server/src/utils/Filter.js

```diff
@@ -1,6 +1,11 @@
 const debug = require('debug')('apphub:server:utils:Filter')
 const Joi = require('./CustomJoi')
-const { parseFilterString, toSQLOperator } = require('./filterUtils')
+const {
+    parseFilterString,
+    toSQLOperator,
+    isVersionOperator,
+    allOperatorsMap,
```

Comment:
why not import `versionOperatorMap` if you're just using this within the body of the applyVersionFilter method?